tour/beerDEcoded.yaml

```diff
@@ -0,0 +1,188 @@
+      uses the minimizer method to sample the k-mers (all the read’s
+      subsequences of length k) in a deterministic fashion in order to reduce
+      memory constumption and processing time. In addition, it masks
+      low-complexity sequences from reference sequences by using dustmasker.<br>
+    backdrop: true
+  - title: <b>Taxonomic classification with Kraken2</b>
+    element: .toolMenuContainer
+    intro: Available tools appear here in the <b>tool menu</b>.
+    position: right
+  - title: <b>Taxonomic classification with Kraken2</b>
+    element: '#__BVID__106'
+    content: >-
+      You can use <b>'tool search'</b> to locate tools.<br><br> Search for 
+      <b>'Kraken2'</b> and select it.<br><br> Tools may take a couple of moments
+      to load.
+    placement: right
+  - title: <b>Taxonomic classification with Kraken2</b>
+    element: .toolMenuContainer
+    intro: >-
+      Open <b>'Kraken2'</b> tool<br><br> Click <b>'Next'</b> to continue our
+      tour.
+    position: right
+  - title: <b>Assign taxonomic labels with Kraken2</b>
+    element: 'div[tour_id="use_names"]'
+    intro: Have a look at the tool's parameters of the <b>'Kraken2'</b> tool.
+    placement: right
+  - title: <b>Assign taxonomic labels with Kraken2</b>
+    intro: >-
+      Please select the following parameters:<br><br> <b>'Single or paired
+      reads'</b>: Single <br> <b>'Input sequences'</b>: Uploaded dataset<br>
+      <b>'Print scientific names instead of just taxids'</b>: Yes<br>
+      <b>'Confidence'</b>: 0.0<br> <b>'In “Create Report”'</b>:<br>
+      &nbsp;&nbsp;&nbsp;&nbsp;<b>'Print a report with aggregrate counts/clade to
+      file'</b>: Yes<br> &nbsp;&nbsp;&nbsp;&nbsp;<b>'Format report output like
+      Kraken 1’s kraken-mpa-report'</b>: Yes<br> <b>'Select a Kraken2
+      database'</b>: fungi2019-03' <br><br>Click <b>'Next'</b> and the tour will
+      <b>'Execute'</b> the Kraken2 tool for you.'
+    postclick:
+      - '#execute'
+  - title: <b>Assign taxonomic labels with Kraken2</b>
+    element: '#current-history-panel > div.controls'
+    intro: >-
+      Congratulations, you have created two files. It contains Classification
+      and Report file.<br><br> It will remain stored in your history.<br><br>
+      Click the <b>'eye'</b> icon to view the data once the history item turns
+      green.
+    position: left
+  - title: <b>Analyze taxonomic assigment</b>
+    content: >-
+      Once we have assigned the corresponding taxa to the sequence, the next
+      step is to properly visualize the data, for which we will use the Krona
+      pie chart tool (<a href="https://doi.org/10.1186/1471-2105-12-385">Ondov
+      et al. 2011</a>).<br>
+    backdrop: true
+  - title: <b>Adjust dataset format</b>
+    element: '#__BVID__106'
+    intro: >-
+      But before that, we need to adjust the format of the data output from
+      Kraken2.<br><br> Search and select the <b>'Reverse'</b> tool.
```

Comment:
> add: It can happen, that the output format of the tool needs to be changed in order for the next tool to read the data. Galaxy offers several "manipulation" tools.

done

Reply:
> It could help to explain what format changes should happen, so they understand why they are using the tools.

I added the info about format changes we need and info about what info kraken2 output contains.